src/app/modules/myinfo/myinfo.adapter.ts

```diff
@@ -269,8 +269,8 @@ export class MyInfoData {
         const data = this.#personData[attr]
         return (
           !!data &&
-          !data.unavailable &&
-          data.source === MyInfoSource.GovtVerified
+          data.source === MyInfoSource.GovtVerified &&
```

Comment:
You know, I suspect some of these fields will only ever be government verified e.g. HDB

Reply:
we could expand further on that in the SDK in the future

Reply:
I think we've discussed this before right, and concluded that we should rely on the data itself as the source of truth? imo it's best not to make assumptions about what may or may not always be government verified. in fact, maybe we should have applied the `MyInfoNotApplicable` type to all the fields instead of only the select ones. any fields may potentially be `NotApplicable` in future.

Reply:
Don't get me wrong, I also think it's better to practice defensive typing in the absence of knowledge. But what I'd like to do also is encode specific domain knowledge into the type system over time so that it serves as a form of documentation.